docs/classification/workflow.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+parent: Classification
+title: Pigeon CLI Workflow
+nav_order: 3
+---
+
+## CLI Workflow
+
+### Map reads to a reference genome
+
+Map reads using _pbmm2_.
+
+```
+pbmm2 align --preset ISOSEQ --sort <input.bam> <ref.fa> <mapped.bam>
+```
+
+### Collapse transcripts into unique isoforms
+
+Collapse mapped reads into unique isoforms using _isoseq collapse_.
```

Comment:
change to "collapse redundant transcripts into unique isoforms based on exonic structures using _isoseq collapse_."